pandas/_libs/tslib.pyx

```diff
@@ -1148,7 +1148,8 @@ cdef class _Timestamp(datetime):
 
     cpdef datetime to_datetime(_Timestamp self):
         """
-        DEPRECATED: use :meth:`to_pydatetime` instead.
+        .. deprecated:: 0.19.0
+           Use :meth:`to_pydatetime` instead.
```

Comment:
It seems sphinx only parses the first line to the summary table as expected. The changes have been reverted to avoid losing information in the API summary table.

![screenshot from 2017-06-15 09-41-54](https://user-images.githubusercontent.com/10288252/27187953-991b8eea-51ba-11e7-9829-fb3d9808940e.png)